esx_service/vmdk_ops.py

```diff
@@ -841,23 +853,26 @@ def main():
     try:
         port = 1019
         opts, args = getopt.getopt(sys.argv[1:], 'hp:')
-    except getopt.error, msg:
+    except getopt.error as msg:
         if msg:
-            print >> sys.stderr, msg
+           logging.exception(msg)
         usage()
-        return 1
+        return(1)
     for a, v in opts:
         if a == '-p':
             port = int(v)
         if a == '-h':
             usage()
-            return 0
+            return(0)
```

Comment:
Nit: I find it somewhat weird to wrap a return statement in parens, but it's no big deal